### PR TITLE
Testing/results and home

### LINKED ISSUE #56: Write tests for Home component

### LINKED ISSUE #73: Write tests for Results component

### LINKED ISSUE #74: Write tests for Card component

### PR DESCRIPTION
**Pull request by ErinUntermeyer**
# Description
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
- I cleaned up some TS errors in multiple test files
- I wrote tests in App, Home, Results and Card
- Closes #56 Closes #73 Closes #74 
## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other, Please explain:

## Has This Been Tested?
- [ ] No
- [x] Yes
- Currently, all tests pass in App, Home, Results and Card

# Message for reviewer:
- If y'all see any other display that could be tested, let me know!

# Checklist:
- [x] My code has no unused/commented out code
- [x] My code has no console logs
- [x] I have performed a self-review of my own code
- [x] I have commented on my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/Card/Card.test.tsx [hunk 1]
```diff
@@ -1,25 +1,65 @@
...
-      <MemoryRouter>
-        <Card
-          name="Las Vegas" 
-        />
-      </MemoryRouter>
-    )
-
-    const cardHeader = screen.getByRole('heading', {name: /las vegas/i})
-
+  it('Should render with all the correct data', () => {
```
Comment (leighlars) [inline src/Card/Card.test.tsx:8]: Looks good! Super fast

File: src/Home/Home.test.tsx [hunk 2]
```diff
@@ -7,39 +7,26 @@ import { getHomeData } from '../helpers/dataFilter'
 jest.mock('../helpers/dataFilter')
```
Comment (leighlars) [inline src/Home/Home.test.tsx:8]: I'll follow suit for Home

File: src/Results/Results.test.tsx [hunk 3]
```diff
@@ -1,13 +1,13 @@
 import React from 'react'
 import Results from './Results'
-import { render, screen } from '@testing-library/react'
+import { render, screen, fireEvent } from '@testing-library/react'
 import { MemoryRouter } from 'react-router-dom'
```
Comment (leighlars) [inline src/Results/Results.test.tsx:4]: Good job

File: src/Results/Results.test.tsx [hunk 4]
```diff
@@ -78,12 +68,34 @@ describe('Results', () => {
 	})
 
 	it('Should filter only results from United States', async () => {
+		const mockedWorldResults = [
```
Comment (JoshSevy) [inline src/Results/Results.test.tsx:71]: These tests look great @ErinUntermeyer. Great work!

**Review by leighlars — APPROVED** [review]
Nice work! I thought there was more to the Testing for TS, but you make it look so easy.

**Review by JoshSevy — APPROVED** [review]
Great Work! The tests look great!

**Pull request merged**